schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -4243,15 +4243,15 @@ FluidLoop:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: True
```

Comment:
False because CHW system could be primary-secondary for one rotation and primary only for another.